src/get-started/index.md.njk

```diff
@@ -8,3 +8,7 @@ show_subnav: true
 This section will enable you to get up and running as quickly as possible. Find easy access to toolkits, guides and helpful information on how to get setup, to start designing at HM Land Registry.
 
 To make your service consistent with GOV.UK, you should start by checking what exists in the [GOV.UK Design System](https://design-system.service.gov.uk).
+
+## Feedback
+
+Is there something missing from this section that you would find useful? Contact HM Land Registry DesignOps. Your feedback will help us to improve it.
```

Comment:
Also, change Contact HM Land Registry DesignOps to "Contact HMLR-Design-System-Support@landregistry.gov.uk."

Reply:
What about linking to the "get in touch" page? https://hmlr-design-fix-duplica-zhyepi.herokuapp.com/get-in-touch/

Reply:
yep, that's perfect

Reply:
Done